components/director/pkg/jsonschema/validator.go

```diff
@@ -0,0 +1,35 @@
+package jsonschema
+
+import (
+	"github.com/xeipuuv/gojsonschema"
+)
+
+type validator struct {
+}
+
+func NewValidator() *validator {
+	return &validator{}
+}
+
+func (v *validator) Validate(jsonschema string, jsons ...string) (bool, error) {
```

Comment:
1. Validate should validate only one json - thanks to that error handling will be easier. 
2. Constructor for validator can accept jsonschema - thanks to that schema will be compiled only once

3. Think about the situation when user does not provide schema - then Validation should always succeed.

Reply:
done